File: Assignment 12/Tony Gradis C++/14.14 Parking Ticket Simulator/Code/ParkedCar.cpp
#include "ParkedCar.h"

ParkedCar::ParkedCar(const string& carMake, const string& carModel, const string& carColor, const string& license, int minutesParked)
    : make(carMake), model(carModel), color(carColor), licenseNumber(license), parkedMinutes(minutesParked) {}

string ParkedCar::getMake() const {
    return make;
}

string ParkedCar::getModel() const {
    return model;
}

string ParkedCar::getColor() const {
    return color;
}

string ParkedCar::getLicenseNumber() const {
    return licenseNumber;
}

int ParkedCar::getParkedMinutes() const {
    return parkedMinutes;
}

File: Assignment 12/Tony Gradis Java/8.8 Parking Ticket Simulator/Code/ParkedCar.java
package Code;

public class ParkedCar {
    private String make;
    private String model;
    private String color;
    private String licenseNumber;
    private int parkedMinutes;

    public ParkedCar(String make, String model, String color, String licenseNumber, int parkedMinutes) {
        this.make = make;
        this.model = model;
        this.color = color;
        this.licenseNumber = licenseNumber;
        if (parkedMinutes >= 0) {
            this.parkedMinutes = parkedMinutes;
        } else {
            this.parkedMinutes = 0;
        }
    }

    public String getMake() {
        return make;
    }

    public String getModel() {
        return model;
    }

    public String getColor() {
        return color;
    }

    public String getLicenseNumber() {
        return licenseNumber;
    }

    public int getParkedMinutes() {
        return parkedMinutes;
    }
}